scout/server/config.py

```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
 SECRET_KEY = "this is not secret..."
-REMEMBER_COOKIE_NAME = "scout_remember_me"
+REMEMBER_COOKIE_NAME = "scout_remember_me"  # Prevent session timeout when user closes browser
+# SESSION_TIMEOUT_MINUTES = 60  # Minutes of inactivity before session times out
```

Comment:
👍

Reply:
Should we fix our config files you think? And how much time do we set?

Reply:
Yes. But possibly wait to deploy that until the personal event log PR is merged?

On the order of 100h should more or less not be noticed. Also anything on the order of 10h should be rather hard to argue against. Setting it to anything below 1h will have an impact on how one can work with open tabs, returning to tasks or showing others partial results. 

How about 10h for starters then? It may force a kind of wrap-up for the day in the evening. But having a recent log to bounce back to would lessen the need for open tabs and the like?

Reply:
> Yes. But possibly wait to deploy that until the personal event log PR is merged?

I think it's more the other way around, that's why I wrote this PR. Checking #3314 I've realized that if you are logged out and you log in you don't get to see the events panel, but you are immediately redirected to the instituteS page. I've commented on that PR that I think the events panel should be moved to some other place. Perhaps you could take also a look at that PR to see what I mean?

Reply:
> How about 10h for starters then?

10 h sounds good. It will mean login every day